packaging/fedora-25/snap-mgmt.sh

```diff
@@ -89,5 +89,5 @@ if [ "$1" = "purge" ]; then
     rm -rf /var/lib/snapd/seccomp/profiles/*
     rm -rf /var/lib/snapd/device/*
     rm -rf /var/lib/snapd/assertions/*
-
+    rm -rf /var/lib/snapd
```

Comment:
Please don't do this here. Do this outside of the script, in your invocation script.